docs/hardware_setup.md

```diff
@@ -131,6 +139,10 @@ To build JetBot you'll need the following tools
 
 ### Step 8 - Attach WiFi module to developer kit
 
+!!! check ""
```

Comment:
I think we can assume B.O.M will guide users on the choice between wifi module for 2GBvs.4GB.  So here we likely just need to mention USB wiFi.

!!! attention ""

    Skip this step if you are using a USB WiFi dongle instead of the M.2 WiFi module